conda.yaml

```diff
@@ -2,8 +2,8 @@ name: tutorial
 channels:
   - defaults
 dependencies:
-  - numpy=1.14.3
-  - pandas=0.22.0
+  - numpy>=1.14.3
```

Comment:
This `numpy>=` dependency change prevents `numpy` conflicts with newer versions of Pandas